reporter/wingtips/src/main/java/zipkin/reporter/wingtips/WingtipsSpanReporter.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2016 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.reporter.wingtips;
+
+import com.nike.wingtips.zipkin.util.ZipkinSpanSender;
+import java.io.Closeable;
+import java.io.IOException;
+import zipkin.Span;
+import zipkin.reporter.AsyncReporter;
+import zipkin.reporter.Encoder;
+import zipkin.reporter.Reporter;
+import zipkin.reporter.Sender;
+
+/** Example bridge between Wingtips' ZipkinSpanSender and this project's reporter */
+public class WingtipsSpanReporter implements ZipkinSpanSender, Reporter<Span>, Closeable {
+  final AsyncReporter<Span> delegate;
+
+  public WingtipsSpanReporter(Sender<byte[]> sender) {
+    this.delegate = AsyncReporter.builder(sender).build(Encoder.THRIFT_BYTES);
```

Comment:
one limitation is it prevents the "toBuilder()" pattern from working when we do it like this... I'm not terribly concerned about this as I don't think people will be boilerplating async reporters.